lib/brew.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import os
+from shutil import which
+
+from devenv.lib import fs
+from devenv.lib import proc
+
+repo_path = "/opt/homebrew"
+
+
+def install() -> None:
+    if which("brew") == f"{repo_path}/bin/brew":
+        return
```

Comment:
Can you add a comment about this check.

I.e. does this mean than brew bundle has been run before?

Reply:
done. no, it's just concerned with installing brew